src/component/teacher/Teacher.js

```diff
@@ -0,0 +1,83 @@
+import { useState, useEffect } from "react";
+import RenderLesson from "./RenderLesson";
+import Form from "./Form";
+import AddQuestion from "./AddQuestion";
+import axios from "axios";
+
+
```

Comment:
Delete line 7, I only want one space between import and function.

Reply:
Thank you @douglas86. Done!